charts/camunda-platform-8.8/values.yaml

```diff
@@ -2568,8 +2568,16 @@ orchestration:
       backoffLimit: 6
       ## @param orchestration.migration.identity.resourceAuthorizationsEnabled if true, the migration of Management Identity data to the orchestration cluster will not create wildcard resource access for roles, this is required to be true if Resource Based Access was used with 8.7
       resourceAuthorizationsEnabled: false
-      ## @param orchestration.migration.identity.idpApplicationId must be define the application ID if an external IdP is used.
-      idpApplicationId: ""
+      ## @param orchestration.migration.identity.clientId must be the clientId used by the migration to access Management Identity. Only needed if external IdP is used (requires read permission).
+      clientId: ""
+      ## @extra orchestration.migration.identity.secret configuration to provide the identity migration client secret.
+      secret:
+          ## @param orchestration.migration.identity.secret.inlineSecret can be used to provide the client secret as a plain-text value for non-production usage.
+          inlineSecret: ""
+          ## @param orchestration.migration.identity.secret.existingSecret can be used to reference an existing Kubernetes Secret containing the client secret.
+          existingSecret: ""
+          ## @param orchestration.migration.identity.secret.existingSecretKey defines the key within the existing secret object.
+          existingSecretKey: ""
```

Comment:
inlineSecret, existingSecret, and existingSecretKey are not referenced in this diff, yet it is added in values.yaml.

Reply:
Nevermind, it is referenced actually.